### PR TITLE
Add PR templates

### PR DESCRIPTION
**Pull request by renatoruk**
Added:
- frontend template (React)
- backend template
- guide how to setup

### CODE REVIEW

File: playbooks/pull-request-templates/pull-request-backend.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Pull request template backend
+description: PR Template Backend
+---
+
+## Jira issue
+
+Url to task.
+
```
Comment (markopaden) [inline playbooks/pull-request-templates/pull-request-backend.md]:
```suggestion
[Jira issue](*link*)
```

File: playbooks/pull-request-templates/pull-request-backend.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+## Summary
+
+Short summary what this PR adds to the project.
+
+## Notes for the reviewer
+
+Note the most important parts that have changed so the reviewer knows what to look.
+
+## How was it tested
+
```
Comment (markopaden) [inline playbooks/pull-request-templates/pull-request-backend.md]:
```suggestion
-   [ ] Postman tests (manual)
```
add this row, it's a good reminder also to test manually.

**Comment by markopaden** [comment]
@renatoruk I would merge this, I already copied backend template into Roltek.

**Comment by renatoruk** [comment]
@markopaden Cool, I'm all for it. I committed your latest proposal. I'll just wait for the CI to complete and then merge.

**Comment by markopaden** [comment]
I added backend template to our project.

**Pull request merged**